### ISSUE TITLE
Restrict the number of data points exported to mlflow.

### ISSUE THREAD

**Issue by tjhunter** [ecmwf/WeatherGenerator#1859]
### Describe the task. Describe the task. It can be a feature, a set of experiments, documentation, etc.

We are pushing a large number of experiments (see internal dashboard), and we have a large space of metric names (source x channel x loss function). We are very close to the official maximum limit of number of metrics per run already. 

The goal of the task: have a whitelist mechanism to push metrics to mlflow for specific values only (the rest of the metrics can be retrieved on the HPC).

- [ ] reach consensus on which metrics to push (configurable)
- [ ] proposal on how to whitelist the metrics
- [ ] implementation

Monitoring is already in place.

### Area

- [ ] datasets, data readers, data preparation and transfer
- [ ] model
- [ ] science
- [x] infrastructure and engineering
- [ ] evaluation, export and visualization
- [ ] documentation

**Comment by clessig** [comment]
We should use for this the headline metrics used already in the forecast evaluation. This works well for ERA5 only training or when ERA5 is present. For other streams, I would suggest to at least always push Stream.mse. We should also always push the overall training loss.

**Comment by pierluigicosi** [comment]
assign me
